etc/cron.d/0800-lokalbolig

```diff
@@ -0,0 +1,2 @@
+# Dump is made 5:30
```

Comment:
Seems  /opt/jobs/general-d8/reset-admin-pass.sql would do just fine instead of opt/jobs/lokalbolig/datadir-preinit-sanitize.sql ?

Reply:
@danquah Good point. Fixed :+1: